src/graphics.rs

```diff
@@ -2,13 +2,17 @@ use std::path::PathBuf;
 
 use wgpu::{Device, Queue};
 
-use client::Client;
-use resources::*;
+use {
+    client::Client,
+    pipeline::Pipelines,
+    resources::{vertex, *},
```

Comment:
I originally fixed them only for the submodules of `graphics`, but I still prefer the "import all" style because it acts like namespaces and I feel reimporting tokens every time discourages splitting code into multiple files; but I'll change them since you asked.